### ISSUE TITLE
AppImage does not run on Fedora 33

### ISSUE THREAD

**Issue by sobotkami** [ragnar-lodbrok/meow-sql#62]
```
$ ./Linux_MeowSQL_0.4.6-x86_64.AppImage
./Linux_MeowSQL_0.4.6-x86_64.AppImage: /tmp/.mount_Linux_altH80/lib/libcrypto.so.1.1: version `OPENSSL_1_1_1b' not found (required by /lib64/libk5crypto.so.3)
```
```
$ dnf list installed | grep openssl
apr-util-openssl.x86_64                            1.6.1-14.fc33                          @fedora                                                      
compat-openssl10.x86_64                            1:1.0.2o-11.fc33                       @fedora                                                      
openssl.x86_64                                     1:1.1.1h-1.fc33                        @updates                                                     
openssl-devel.x86_64                               1:1.1.1h-1.fc33                        @updates                                                     
openssl-libs.i686                                  1:1.1.1h-1.fc33                        @updates                                                     
openssl-libs.x86_64                                1:1.1.1h-1.fc33                        @updates                                                     
openssl-pkcs11.i686                                0.4.10-7.fc33                          @fedora                                                      
openssl-pkcs11.x86_64                              0.4.10-7.fc33                          @fedora                                                      
rubygem-openssl.x86_64                             2.1.2-135.fc33                         @updates                                                     
xmlsec1-openssl.x86_64                             1.2.29-2.fc33                          @fedora
```

**Comment by ragnar-lodbrok** [comment]
Does previous one run (https://github.com/ragnar-lodbrok/meow-sql/releases/tag/v0.4.5-alpha) ?

**Comment by sobotkami** [comment]
Unfortunately it doesn't run too. 

```
$ ./Linux_MeowSQL_0.4.5-x86_64.AppImage
./Linux_MeowSQL_0.4.5-x86_64.AppImage: /tmp/.mount_Linux_AvLgQT/lib/libcrypto.so.1.1: version `OPENSSL_1_1_1b' not found (required by /lib64/libk5crypto.so.3)
```

Btw, the AppImage of 0.4.6 is as twice big as the the previous one 0.4.5 --> if you extract the AppImage of 0.4.6, the AppImage of 0.4.5 is also there :smile_cat:

**Comment by ragnar-lodbrok** [comment]
I found that this is a known bug of linuxdeployqt: https://github.com/probonopd/linuxdeployqt/issues/391
They suggest to build on Ubuntu 16.04 Xenial, what I did, see another AppImage Linux_MeowSQL_0.4.6-x86_64_xenial.AppImage
However it gives me another error on Fedora, brrr. 
Looks like that I need to replace linuxdeployqt with linuxdeploy to make AppImage.

And yes, I didn't notice it will copy entire folder, so thanks for posting, AppImage is now 49.3 MB.

**Comment by sobotkami** [comment]
First, thank you for updated AppImages :+1: 

Do you mean the second one?
```
$ ./Linux_MeowSQL_0.4.6-x86_64.AppImage 
./Linux_MeowSQL_0.4.6-x86_64.AppImage: /tmp/.mount_Linux_OTmiuN/lib/libcrypto.so.1.1: version `OPENSSL_1_1_1b' not found (required by /lib64/libk5crypto.so.3)

$ ./Linux_MeowSQL_0.4.6-x86_64_xenial.AppImage 
./Linux_MeowSQL_0.4.6-x86_64_xenial.AppImage: symbol lookup error: /lib64/libk5crypto.so.3: undefined symbol: k5_utf8_to_utf16le, version krb5support_0_MIT
```
Which version of Qt do you use in the AppImage? The `xenial` seems to have the `5.5.1`.
Why I ask you: https://github.com/probonopd/linuxdeployqt/issues/209#issuecomment-530998586 
> Qt 5.10 seems to resolve the issue.

TL; DR: I don't have big experiences with AppImage so I'm maybe wrong.

**Comment by ragnar-lodbrok** [comment]
Yeap, I meant the 2nd Linux_MeowSQL_0.4.6-x86_64_xenial.AppImage
I used Qt 5.14 built for xenal from https://launchpad.net/~beineri as they say Qt 5.10+ should work.
Need to dig deeper.

**Comment by sobotkami** [comment]
Just retested with the new version 0.4.8 on Fedora 33 and it works great

**Comment by KUPASH** [comment]
@ragnar-lodbrok Hi! I have a download problem with AppImage in Chrome. I have not found another way to download =(
